### PR TITLE
Rename Charon to BPMN-Studio

### PR DESCRIPTION
**Pull request by SteffenKn**
## What did you change?

I just renamed Charon to BPMN-Studio as wished in #131.

- 🔜 https://github.com/process-engine/process_engine_meta/pull/23
- 🔜 https://github.com/process-engine/documentation/pull/63
- 🔜 https://github.com/process-engine/consumer_client/pull/13

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# Charon (/ˈkɛərɒn/)
+# BPMN-Studio
 
 ProcessEngineJS Admin Panel
 
-> Charon is the ferryman of Hades who carries souls of the newly deceased across the rivers Styx and Acheron that divided the world of the living from the world of the dead.
+> BPMN-Studio is the ferryman of Hades who carries souls of the newly deceased across the rivers Styx and Acheron that divided the world of the living from the world of the dead.
```
Comment (Vyperus) [inline README.md]: This is unneccessary now.

File: package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@process-engine/charon",
+  "name": "@process-engine/bpmn-studio",
```
Comment (Vyperus) [inline package.json]: Pls delete the scope here.

File: package.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@process-engine/charon",
+  "name": "bpmn-studio",
   "description": "The process-engine frontend.",
```
Comment (alexanderkasten) [inline package.json]: I thought the BPMN-Studio has nothing to do with the process engine?

File: README.md [hunk 4]
```diff
@@ -1,14 +1,10 @@
-# Charon (/ˈkɛərɒn/)
+# BPMN-Studio
 
 ProcessEngineJS Admin Panel
```
Comment (Vyperus) [inline README.md]: This should be deleted as well

File: README.md [hunk 5]
```diff
@@ -1,14 +1,10 @@
...
 
-> Charon is the ferryman of Hades who carries souls of the newly deceased across the rivers Styx and Acheron that divided the world of the living from the world of the dead.
-
-It's basically a frontend
-
 ## Was sind die Ziele dieses Projekts?
 
-Charon soll die bestehende React Anwendung ablösen und als zentrale Verwaltungsschnittstelle für die ProcessEngine dienen.
+BPMN-Studio soll die bestehende React Anwendung ablösen und als zentrale Verwaltungsschnittstelle für die ProcessEngine dienen.
```
Comment (Vyperus) [inline README.md]: Maybe we should replace this by writing a little sentence about BPMN-Studio.

File: README.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
 # BPMN-Studio
 
-ProcessEngineJS Admin Panel
-
 ## Was sind die Ziele dieses Projekts?
 
-BPMN-Studio soll die bestehende React Anwendung ablösen und als zentrale Verwaltungsschnittstelle für die ProcessEngine dienen.
+BPMN-Studio kann verwendet werden um BPMN Diagramme zu erstellen.
+Des Weitern kann BPMN-Studio mit einer Workflow Engine verbunden werden,
```
Comment (Vyperus) [inline README.md]: Des Weiteren...

File: README.md [hunk 7]
```diff
@@ -1,10 +1,10 @@
 # BPMN-Studio
 
-ProcessEngineJS Admin Panel
-
 ## Was sind die Ziele dieses Projekts?
 
-BPMN-Studio soll die bestehende React Anwendung ablösen und als zentrale Verwaltungsschnittstelle für die ProcessEngine dienen.
+BPMN-Studio kann verwendet werden um BPMN Diagramme zu erstellen.
```
Comment (Vyperus) [inline README.md]: Evtl. so ?: 
BPMN-Studio soll es dem Anwender so leicht wie möglich machen BPMN-Diagramm zu erstellen und zu pflegen.

File: README.md [hunk 8]
```diff
@@ -1,14 +1,11 @@
...
-
-> Charon is the ferryman of Hades who carries souls of the newly deceased across the rivers Styx and Acheron that divided the world of the living from the world of the dead.
-
-It's basically a frontend
+# BPMN-Studio
 
 ## Was sind die Ziele dieses Projekts?
 
-Charon soll die bestehende React Anwendung ablösen und als zentrale Verwaltungsschnittstelle für die ProcessEngine dienen.
+BPMN-Studio soll es dem Anwender so leicht wie möglich machen BPMN-Diagramm
```
Comment (heikomat) [inline README.md]: Typo: `e` fehlt bei BPMN-Diagramme

File: bin/bpmn-studio.js [hunk 9]
```diff
@@ -11,7 +11,7 @@ config.contentType.ttf = 'application/x-font-ttf';
 
 server.deploy(config, (result) => {
   const url = `http://localhost:${result.config.port}/`;
-  console.log(`Charon started at ${url}`);
+  console.log(`BPMN-Studio started at ${url}`);
```
Comment (cmg-dev) [inline bin/bpmn-studio.js]: Please like this:

```
console.log(`BPMN-Studio started at: '${url}'`);
```

File: README.md [hunk 10]
```diff
@@ -1,14 +1,11 @@
-# Charon (/ˈkɛərɒn/)
-
-ProcessEngineJS Admin Panel
-
-> Charon is the ferryman of Hades who carries souls of the newly deceased across the rivers Styx and Acheron that divided the world of the living from the world of the dead.
-
-It's basically a frontend
+# BPMN-Studio
```
Comment (cmg-dev) [inline README.md]: > BPMN-Studio ist eine Web-Applikation zur Erstellung, Verwaltung, Ausführung und Auswertung von BPMN-Prozessen.
> 
> Es setzt auf dem BPMN.io auf und den BPMN-Standard 2.x um.

Comment (cmg-dev) [reply README.md]: Bitte den Text noch hierhin stopfen.

**Pull request merged**